src/paas_charm/app.py

```diff
@@ -26,7 +26,7 @@
     from paas_charm.redis import PaaSRedisRelationData
     from paas_charm.s3 import PaaSS3RelationData
     from paas_charm.saml import PaaSSAMLRelationData
-    from paas_charm.tempo import PaaSTempoRelationData
+    from paas_charm.tracing import PaaSTracingRelationData
```

Comment:
Renamed `tempo` to `tracing` to reflect integration business logic rather than capturing the technology behind it.